hummingbot/core/gateway/__init__.py

```diff
@@ -7,6 +7,7 @@
 
```

Comment:
Since we are removing `gateway create`, this function can be made much simpler, since only `local_certs_path` is needed. The other 5 elements in the mapping are never used.

Consider refactoring this function either in this PR or in the removal Gateway one. I changed it to the following function in this test branch: https://github.com/hummingbot/hummingbot/blob/feat/certs/hummingbot/core/gateway/__init__.py

```
def get_certs_path(client_config_map: "ClientConfigAdapter") -> Path:
    """
    Calculates the default cert paths for a gateway container.
    """
    if is_inside_docker():
        return os.getenv("CERTS_FOLDER") and Path(os.getenv("CERTS_FOLDER"))
    else:
        return Path(client_config_map.certs.path)
```

Reply:
I can't see where is this code, or which method we should change. But based on your previous pr, I didn't remove this because it has some logic to use ENV environments and creating the folders if they don't exist

Reply:
ok we can leave it in as-is